### PR TITLE
fix CI issues

### PR DESCRIPTION
**Pull request by andrei-ng**
- fix clippy suggestions 
 - change workflow to use new 'main' branch 
 - disable failing tests for windows, to be fixed in https://github.com/plotly/plotly.rs/issues/192

### CODE REVIEW

File: plotly_kaleido/src/lib.rs [hunk 1]
```diff
@@ -163,7 +163,7 @@ impl Kaleido {
         }
 
         let output_lines = BufReader::new(process.stdout.unwrap()).lines();
-        for line in output_lines.flatten() {
+        for line in output_lines.map_while(Result::ok) {
```
Comment (ndrezn) [inline plotly_kaleido/src/lib.rs:166]: Doesn't look related to issue title but plausibly just adding in a stray cleanup?

File: examples/statistical_charts/src/main.rs [hunk 2]
```diff
@@ -345,7 +345,7 @@ fn fully_styled_box_plot() {
         v
     };
 
-    let x_data = vec![
+    let x_data = [
```
Comment (ndrezn) [inline examples/statistical_charts/src/main.rs:348]: Do we just not need to declare `vec!` here?

File: plotly_kaleido/src/lib.rs (hunk 1 above)
Comment (andrei-ng) [reply plotly_kaleido/src/lib.rs:166]: Yes,  you are correct. Updated the title and added a bit more context to the PR description.

Rust comes with [`clippy`](https://doc.rust-lang.org/clippy/) , a very nice linter. The latest clippy introduced two more lints which were fixed in the second commit of this PR.

File: examples/statistical_charts/src/main.rs (hunk 2 above)
Comment (andrei-ng) [reply examples/statistical_charts/src/main.rs:348]: This comes from a `clippy` lint .

**Review by ndrezn — APPROVED** [review]
💃
